.travis.yml

```diff
@@ -6,13 +6,10 @@ cache:
 notifications:
   email: false
 node_js:
-  - stable
-  - '0.10'
-before_install:
-  - npm i -g npm@^3.0.0
+  - 'node'
```

Comment:
`src` code will not pass in `node 4` as it does not support all the `es-next` syntax. Anyhow, for `node 4` et al, the dis code need to be tested. I am not sure if that is the scope of this PR.